### ISSUE TITLE
IDEA - Add input box next to sliders

### ISSUE THREAD

**Issue by dmahony** [ilia3101/MLV-App#207]
Adding an input box next to the sliders will allow more fine tuned controls, is this difficult to implement?

**Comment by masc4ii** [comment]
Thanks for the request. But the feature already exists.

**Comment by dmahony** [comment]
Well I feel silly haha. It could be more obvious though, perhaps a box around the values?

**Comment by ilia3101** [comment]
Maybe cursor could become the text location one when hovering over value?

(Boxes might look ugly)

**Comment by masc4ii** [comment]
Yes. Boxes look really ugly. I tried to paint the background lightgray, similar to Lightroom. But our layout realization is not made for this, so this looks very ugly too. I then tried to color the text itself to orange - same like the active clip in the session when not selected. This looks okay. So I commited this small change.

(Exposure hovered:)
![Bildschirmfoto 2020-06-15 um 20 29 42](https://user-images.githubusercontent.com/30245296/84692945-fae36700-af46-11ea-934d-2b27b7f7d5ed.png)
